app/src/components/markers/priceTagMarker.tsx

```diff
@@ -63,13 +65,21 @@ export class PriceTagMarker extends Component<MarkerProps, flightSearchParameter
     }
 
     componentDidMount(): void {
+        this.onHover();
         this.addListener();
     }
 
     componentDidUpdate(): void {
         this.addListener();
     }
 
+    onHover(): void {
+        let tag = findDOMNode(this) as Node;
+        tag.addEventListener('click', () => {
```

Comment:
just wonder, is it really should be 'click' eventlistener to fire hover event?? click->hover?

Reply:
We are using custom markers and the lat long which we have is not precisely the same as what google maps would return on click/hover. Not sure if we can have a line on hover using custom marker, that is why I put click.

Reply:
can you send a screenshot with the example? but if must show the line on hover, is that how it works?

Reply:
maybe it is possible, but we need to handle events on the map, I am not sure how to handle a hover event on the marker and correspond it to a hover event on the map. U need a screenshot of what we have right now?

Reply:
'lat long which we have is not precisely the same as what google maps would return on click/hover'.
If it show a line on Click, but not on hover, we won't merge it with dev, sorry. 
what if you change it to tag.addEventListener('**hover**', ...)